overlord/servicestate/resources/quotas.go

```diff
@@ -0,0 +1,105 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
actually is there a circular import problem that prevents us from putting all this inside `snap/quota` directly?

Reply:
Exactly, the reason it goes in resources/ folder is to solve a circular import problem. I'll just try to see if it would also exist if i put it directly into snap/quota.